app/src/main/res/drawable/ic_list.xml

```diff
@@ -0,0 +1,7 @@
+<vector android:height="24dp" android:viewportHeight="129.0"
```

Comment:
WARNING: [UnusedResources] The resource `R.drawable.ic_list` appears to be unused
 Unused resources make applications larger and slow down builds.